src/arm64le.h

```diff
@@ -36,32 +36,40 @@
 #define DES_ASM				0
 #define DES_128K			0
 #define DES_X2				0
-#define DES_MASK			0
+#define DES_MASK			1
 #define DES_SCALE			1
 #define DES_EXTB			0
-#define DES_COPY			1
+#define DES_COPY			0
 #define DES_BS_ASM			0
-#define DES_BS				1
 #if __ARM_NEON && !JOHN_NO_SIMD
```

Comment:
What would define `__ARM_NEON` (checked here and in one place below)? It is _not_ defined by the compiler on AArch64, at least not in my tests. Instead, we could check for `__aarch64__`, but there's no need to do that in this file since our usage of this file implies we're on that arch.